index.html

```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/3085842107.js" crossorigin="anonymous"></script>
+    <script defer src="node_modules/bootstrap/dist/js/bootstrap.bundle.js"></script>
+    <link rel="stylesheet" href="dist/css/main.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="/dist/css/main.css">
```

It's better to use root links like this so that you have an easier time when your html structure grows